controller.yml

```diff
@@ -44,3 +44,12 @@
      - python-zmq
      - apache2
   - include: controller_extension.yml
+
+- hosts: cc*
```

Comment:
Again please create a separate file to be used again as per will and use include.